src/commands/entities/configure.ts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as EntitiesAPI from '../../mix/api/entities'
+import * as MixFlags from '../../utils/flags'
+import {EntitiesConfigureParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+import {DomainOption} from '../../utils/validations'
+import {validateRegexEntityParams, validateRuleBasedEntityParams} from '../../utils/validations'
+
+const debug = makeDebug('mix:commands:entities:configure')
+
+export default class EntitiesConfigure extends MixCommand {
+  static description = `configure an entity
+
+Use this command to configure an entity in a project.
+
+NOTE: This command cannot change the type of an entity.
+When using this command, you must provide the current type
+of the entity. Use command entities:convert to change the
```

Comment:
What a UX pain. Not our 'fault', of course, but it's quite irritating that users need to supply a type just to get the command to work. I'm almost tempted to suggest an `--entity-type=auto` mode, which GETs the project behind the scenes to secure the type.

Reply:
Yes, that would indeed require a second endpoint call and also bring up other usability issues. For instance, there would need to be error handling around trying to update a base entity, which is something you cannot get to currently as --entity-type does not accept 'base' as a possible value. We are going to proceed with having users provide the --entity-type flag but took note of it.